assignment2/lib/storage/persistant_storage.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:convert';
+import 'package:assignment2/models/user.dart';
+import 'package:assignment2/storage/session_storage.dart';
+import 'dart:io';
+
+class PersistantStorage {
+  //singleton class
+  static PersistantStorage? _instance;
+  static PersistantStorage get instance => _instance ??= PersistantStorage._();
+  PersistantStorage._();
+
+  List<User> userList = SessionStorage.instance.getuserDetails();
+
+  String _returnJsonString(List<User> userList) {
+    var userListMap = <String, String>{};
+    userList.forEach((user) {
+      userListMap[user.rollNo.toString()] = jsonEncode(user.toJson());
+    });
+    return jsonEncode(userListMap);
+  }
+
+  void save(List<User> userList) {
+    String userListMapInString = _returnJsonString(userList);
+    File('C://Users//acer//Desktop//Learning projects//git//Nuclei-assignments//assignment2//lib//storage//persistant_storage_file.txt')
```

Comment:
Store the path in a constant.. also provide relative path here so that it can run on other systems as well without any modifications to the path